src/flows/pipes/Visualization/view.tsx

```diff
@@ -69,7 +69,7 @@ const Visualization: FC<PipeProp> = ({Context}) => {
         <div className="flow-visualization">
           <div className="flow-visualization--view">
             <View
-              loading={loading}
+              loading={RemoteDataState.Done}
```

Comment:
should this be `RemoteDataState.Loading`?

Reply:
the View requires a loading state, and the parent handles that loading state spinner, removing the need for it to be handled by the View. The way it was, with both handling the issue, we got two spinners on a visualization

making it Loading here would mean the view is always behind a spinner.

Reply:
ah okay, got it so the intention is to remove the duplicate spinner 👍